tests/tt_metal/tt_metal/api/test_compile_defines.cpp

```diff
@@ -0,0 +1,29 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Nit: can you fix the licence header like so:
```suggestion
// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
//
// SPDX-License-Identifier: Apache-2.0

```